.github/workflows/ci.yml

```diff
@@ -72,5 +76,13 @@ jobs:
         env:
           RAILS_ENV: test
           TEST_DATABASE_URL: ${{ steps.setup-mysql.outputs.db-url }}
-        run: bundle exec rake test
+          CI_NODE_TOTAL: ${{ matrix.ci_node_total }}
+          CI_NODE_INDEX: ${{ matrix.ci_node_index }}
+        run: ./script/minitest-ci
 
+  test-ruby:
+    name: Test Ruby
+    needs: test-ruby-matrix
+    runs-on: ubuntu-latest
+    steps:
+      - run: echo "All Minitest tests have passed 🚀"
```

Comment:
😎

Reply:
I can't take credit for that - I copied it from the Whitehall version!